microsoftgraph/microsoft-graph-explorer-v4
# Fix: Autocollapsing of history items

## Overview
Fixes #2330 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the History tab
Click on any History item in the Yesterday or Older categories
Notice the fix
